website/ref/language.md

```diff
@@ -2392,18 +2393,27 @@ itself or defined by the user.
 
 ### Importing modules with `use`
 
-Modules are imported using the `use` special command, which accepts a **module
-spec** and an optional alias:
+Modules are imported using the `use` special command. It requires a module spec
+and an optional alias:
```

Comment:
Not sure about this change. "Requires ... an optional..." is an oxymoron; "accept" is more appropriate. Also not sure why you'd removed the emphasis on `module spec`; it is the first time this term appears in the documentation and should be emphasized.

Reply:
I settled on this verbiage: "It requires a module spec and allows a namespace alias:". I removed the bolding because it doesn't tell the user anything meaningful but I reinstated it since I, sort of, see your point.